images/generic-autobumper/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM golang:1.20.4-alpine3.17 as builder
```

It looks like the project is [still using v1.18](https://github.com/kubernetes/test-infra/commit/52e11cab1ea80f71e7af5f69573ffbb17b234cca)
```suggestion
FROM golang:1.18 as builder
```